mappings/net/minecraft/world/gen/chunk/BlendingData.mapping

```diff
@@ -56,15 +56,21 @@ CLASS net/minecraft/class_6749 net/minecraft/world/gen/chunk/BlendingData
 		ARG 2 chunkBlockX
 		ARG 3 chunkBlockZ
 		ARG 4 surfaceHeight
+	METHOD method_39567 (D)Z
+		ARG 0 height
 	METHOD method_39570 getBlendingData (Lnet/minecraft/class_3233;II)Lnet/minecraft/class_6749;
 		ARG 0 chunkRegion
 		ARG 1 chunkX
 		ARG 2 chunkZ
+	METHOD method_39571 (Lcom/mojang/serialization/codecs/RecordCodecBuilder$Instance;)Lcom/mojang/datafixers/kinds/App;
+		ARG 0 instance
 	METHOD method_39572 initChunkBlendingData (Lnet/minecraft/class_2791;Ljava/util/Set;)V
 		ARG 1 chunk
 		ARG 2 newNoiseChunkDirections
 	METHOD method_39573 validate (Lnet/minecraft/class_6749;)Lcom/mojang/serialization/DataResult;
 		ARG 0 data
+	METHOD method_39574 ([D)V
+		ARG 0 heights2
```

Comment:
Is this the convention for situations like this?

Reply:
This is a weird situation, but yes we append `2` or `3` in case of conflicts (most often seen as `buf2`).